modules/acquia_cms_common/acquia_cms_common.module

```diff
@@ -490,20 +422,12 @@ function acquia_cms_common_module_implements_alter(array &$implementations, stri
     }
   }
   // Prevent installation of site studio package on module install
-  // during site installation from other modules, this causes issue.
-  if ($hook == 'modules_installed' && InstallerKernel::installationAttempted()) {
-    unset($implementations['cohesion_sync']);
-  }
-}
-
-/**
- * Implements hook_css_alter().
- */
-function acquia_cms_common_css_alter(array &$css) {
-  if (\Drupal::service('module_handler')->moduleExists('collapsiblock')) {
-    // Remove the default css provided by module: `collapsiblock`.
-    $path = \Drupal::service('extension.list.module')->getPath('collapsiblock') . '/theme/dist/css/collapsiblock.min.css';
-    unset($css[$path]);
+  // during site installation & configuration synchronization
```

Comment:
@chandan-singh7929 Add todo here with ACO ticket number mentioning that we need to revert once that ticket is released.

Reply:
Done